### PR TITLE
Migrate data

### LINKED ISSUE #41: Adjust Django schema to follow the API schema
All functionality must be checked and also check that the generated models are following the API's

### LINKED ISSUE #42: Group S: Can't sign up
billede

We get a 403 when trying to sign up on your application.

### LINKED ISSUE #43: Group S: Does not show public simulator tweets
The app only shows four tweets, none of which are from the simulator. 

image

### PR DESCRIPTION
**Pull request by PetraSt**

### CODE REVIEW

**Comment by PetraSt** [comment]
To start the migration of an old DB:

1. delete all the DBs in your folder
2. create an empty DB with name minitwit.db
3. add the old DB to the project (for now it is the minitwitOld.db)
4. python manage.py migrate MiniTwit 0001_user_model
5. python manage.py migrate MiniTwit 0002_message_model
6. python manage.py migrate
7. python manage.py runserver

### **before approving test the above steps and add a user, login, publish a message, logout, login again and if we do not have any issues we can merge** ###

File: ITU_MiniTwit/MiniTwit/forms.py
Comment (szymongalecki) [inline ITU_MiniTwit/MiniTwit/forms.py:1]: Why there are two passwords?

File: ITU_MiniTwit/MiniTwit/migrations/0001_user_model.py [hunk 1]
```diff
@@ -1,6 +1,44 @@
...
+            # Copy the relevant data from the old table to the new table
+            new_user = newUser()
+
+            new_user.id = row[0]
+            new_user.password = row[3]
+            new_user.email = row[2]
+            new_user.username = row[1]
+            new_user.first_name = row[1].split()[0]  # first name is the first word in the username
+            if len(row[1].split()) > 1:
+                new_user.last_name = row[1].split()[1]
```
Comment (szymongalecki) [inline ITU_MiniTwit/MiniTwit/migrations/0001_user_model.py]: We might potentially lose same data here if user is called for example "Maria Sofia Lopez" but I think that the majority of names does have two parts only.

**Review by szymongalecki — APPROVED** [review]
I can see that you put a lot of work into this, code looks good!

**Comment by dprzygocka** [comment]
needs QA

File: ITU_MiniTwit/MiniTwit/forms.py
Comment (PetraSt) [reply ITU_MiniTwit/MiniTwit/forms.py:1]: The second one is for the retyped password.
And to check that the first is equal with the retyped one

**Pull request merged**